apps/mail/app/globals.css

```
@@ -1,156 +1,295 @@
+
+/*
+  Automatic source detection should suffice, but for more control, check
+  the docs on Detecting classes in source files, namely `source(none)`:
+  https://tailwindcss.com/docs/detecting-classes-in-source-files
+*/
+@import 'tailwindcss';
+
+/* Plugins */
+@plugin "tailwindcss-animate";
+@plugin "tailwind-scrollbar" {
+  nocompatible: true;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)